### PR TITLE
feat(DLI): import DLI global variable resource and add unit test and document.

### PR DESCRIPTION
**Pull request by Zippo-Wang**
**What this PR does / why we need it**:
import `resource_flexibleengine_dli_global_variable` resource, unit test and document

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
=== RUN   TestAccGlobalVariable_basic
=== PAUSE TestAccGlobalVariable_basic
=== CONT  TestAccGlobalVariable_basic
--- PASS: TestAccGlobalVariable_basic (231.00s)

PASS

coverage: 1.7% of statements in ../../../terraform-provider-flexibleengine/...
```

### CODE REVIEW

File: docs/resources/dli_global_variable.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) The name of a Global variable.
+  This parameter can contain a maximum of 128 characters, which may consist of digits, letters, and underscores (\_),
+  but cannot start with an underscore (\_) or contain only digits.
+
+  Changing this parameter will create a new resource.
+
+* `value` - (Required, String) The value of Global variable.
```
Comment (chengxiangdong) [inline docs/resources/dli_global_variable.md:31]:
```suggestion
* `value` - (Required, String) The value of global variable.
```

File: flexibleengine/acceptance/resource_flexibleengine_dli_global_variable_test.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+			},
+		},
+	})
+}
+
+func testGlobalVariable_basic(name string) string {
+	return fmt.Sprintf(`
+resource "flexibleengine_dli_global_variable" "test" {
+  name = "%s"
+  value = "abc"
```
Comment (chengxiangdong) [inline flexibleengine/acceptance/resource_flexibleengine_dli_global_variable_test.go:109]:
```suggestion
  name  = "%s"
  value = "abc"
```

File: flexibleengine/acceptance/resource_flexibleengine_dli_global_variable_test.go [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+}
+`, name)
+}
+
+func testGlobalVariable_basic_update(name string) string {
+	return fmt.Sprintf(`
+resource "flexibleengine_dli_global_variable" "test" {
+  name = "%s"
+  value = "abcd"
```
Comment (chengxiangdong) [inline flexibleengine/acceptance/resource_flexibleengine_dli_global_variable_test.go:118]:
```suggestion
  name  = "%s"
  value = "abcd"
```

**Review by chengxiangdong — COMMENTED** [review]
Just some little nits, can you fix them.

File: docs/resources/dli_global_variable.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
+---
+subcategory: "Data Lake Insight (DLI)"
+---
+
+# flexibleengine_dli_global_variable
+
+Manages a DLI global variable resource within FlexibleEngineCloud.
```
Comment (chengxiangdong) [inline docs/resources/dli_global_variable.md:7]:
```suggestion
Manages a DLI global variable resource within FlexibleEngine.  
```

Comment (Zippo-Wang) [reply docs/resources/dli_global_variable.md:7]: from huaweicloud not huawei to FlexibleEngineCloud, keep them consistent

**Pull request merged**